tests/integration/test_kubeapi-load-balancer_integration.py

```diff
@@ -20,22 +20,31 @@ def _check_status_messages(ops_test):
 
 @pytest.mark.abort_on_fail
 async def test_build_and_deploy(ops_test):
+    log.info("Build Charm...")
+    charm = await ops_test.build_charm(".")
+
+    log.info("Build Bundle...")
     bundle = ops_test.render_bundle(
-        "tests/data/bundle.yaml", k8s_lb_charm=await ops_test.build_charm(".")
+        "tests/data/bundle.yaml", k8s_lb_charm=charm
     )
-    await ops_test.model.deploy(bundle)
+
+    log.info("Deploy Bundle...")
+    model = ops_test.model_full_name
+    cmd = f"juju deploy -m {model} {bundle}"
+    rc, stdout, stderr = await ops_test.run(*shlex.split(cmd))
+    assert rc == 0, f"Bundle deploy failed: {(stderr or stdout).strip()}"
```

Comment:
Same question about not deploying the bundle with the pytest operator

Reply:
Same answer :)

it seems `ops_test.deploy_bundle` requires another snap package in the environment called `juju-bundle` which relies on a library `rust-libjuju` which is out of date in supporting any ubuntu series past [eoan](https://github.com/knkski/rust-libjuju/issues/3).

this is the best i've got access to at the moment.